pages/landing/enterprise-eks/contact/_form.html

```diff
@@ -39,42 +39,35 @@ <h1 style="text-align: center;">
       <div class="form-group">
         <div class="row">
           <div class="col-xs-12 col-md-4">
-            <label>Which features are you interested in?</label>
-          </div>
-        </div>
-      </div>
-      <div class="form-group">
-        <div class="row">
-          <div class="col-xs-12 col-md-4">
-            <label>Service Mesh</label>
+            <label style="padding-top: 0;">Service Mesh</label>
```

Comment:
NIT: all the `<label>` elements in the left column need the `padding-top` fix; it looks like you've added it here, but "Monitoring", "App languages / frameworks", and "Security / Compliance" still need it.

Reply:
Done: 7911607